src/modules/core/components/Fields/Input/InputComponent.css

```diff
@@ -35,6 +35,7 @@
   composes: main;
   height: 45px;
   font-size: var(--size-medium);
+  font-weight: var(--weight-bold);
```

Comment:
Please don't change the core component globally for a change we need in a local component.

Please revert this and make the change in the local component's stylesheet

Reply:
But i check inputs in designs and they all seems to have bolder font

Reply:
see edit colony details, input storage, i can check this with design if we plan to be consistent with this across the app

Reply:
Notice also that this is for "fat" theme

Reply:
So when I looked at it on my screen it looked way bigger than it was supposed to be, like size `16xp` or something.

I'll check again.

Reply:
but nvm, i will use it locally

Reply:
Nvm, im changing to --family-techie like you proposed and adding "medium" weight to look nice